src/components/atoms/Toggler/Toggler.scss

```diff
@@ -0,0 +1,67 @@
+.Toggler {
```

Comment:
Another removed file that seems to be making a comeback.

Is this one supposed to be added? 

If so a `.module.scss` version would be nicer and then this one would be moved to `./src/components/attendee/`, but if it is for admin side, then this file, according to how we're moving components around with the redesign, shouldn't even exist, but a `.tailwind.ts` one and moved to `./src/components/admin`

Reply:
moved toggler to `attendee` components + updated styles [8bf8d88](https://github.com/sparkletown/sparkle/pull/2840/commits/8bf8d88aa41daa2e728044cf16a1ace637c3f9db)